integration-tests/mysql_test.go

```diff
@@ -124,8 +124,8 @@ var _ = Describe("MySQL", Label("MySQL"), func() {
 					HaveKeyWithValue("storage_gb", float64(100)),
 					HaveKeyWithValue("storage_type", "io1"),
 					HaveKeyWithValue("iops", float64(3000)),
-					HaveKeyWithValue("storage_autoscale", false),
-					HaveKeyWithValue("storage_autoscale_limit_gb", float64(0)),
+					HaveKeyWithValue("storage_autoscale", true),
+					HaveKeyWithValue("storage_autoscale_limit_gb", float64(250)),
```

Comment:
Just a hint that you can also use the `BeNumerically()` matcher rather than casting to `float64` - although it's a matter of preference.